windows/README.md

```diff
@@ -111,3 +111,15 @@ Set-Item WSMan:localhost\client\trustedhosts -value *
 You can confirm this ran successfully with, `get-item WSMan:\localhost\Client\TrustedHosts`, which should list an output with a `*` under the Value column.
 
 After running this command you should be able to try again and successfully connect to the remote hosts to gather data.
+
+### Server name cannot be resolved error
+
+Run ping \<subject-ipv4\>. If there's no response, you need to take steps to ensure these servers can communicate with each other.
+
+Ensure the security group of the subject VMs allows incoming connections from the controller machine.
+
+Add the subject machines to the TrustedHosts file of the controller - see [Authentication error](#authentication-error).
+
+Ensure the firewall of the subject machines allows connections from the controller device.
```

Comment:
An idea for the guides: In case users want to set the value with restrictions, `trustedhosts` also accepts server names or IP addresses. i.e., `Set-Item wsman:\localhost\client\trustedhosts *.google.com`
([source](https://techdirectarchive.com/2020/03/23/how-to-add-trusted-host-for-the-winrm-client/))